### ISSUE TITLE
Throwing DoctrineBundle exception in HautelookAliceExtension

### ISSUE THREAD

**Issue by ixarlie** [hautelook/AliceBundle#385]
Hi, I'm trying to configure this bundle v2.0.0 in my project that only uses DoctrineMongoDBBundle. The HautelookAliceExtension [is requesting for the ORM bundle](https://github.com/hautelook/AliceBundle/blob/v2.0.0/src/DependencyInjection/HautelookAliceExtension.php#L48) that I don't intend to use at the moment.

If the persistence layer is responsibility of theofidry/alice-data-fixtures, why do the extension check this dependency?

Sorry if I misunderstood anything. Thanks!

**Comment by theofidry** [comment]
Hi, right now HautelookAliceBundle 2.0 supports only Doctrine ORM and the support for MongoDB via DoctrineODM is still missing (see #324).

Also there is some DX issues at the moment that could be solved with a flex recipe (see #380).

**Comment by ixarlie** [comment]
Ok, but it seems like the HautelookAliceExtension v2.0.0 is too tied to a particular persistence layer when that should be totally moved to FidryAliceDataFixturesBundle (loading and saving) because it is in there where I declared my persistence layers.

I miss some configuration to say what persistence layer I want to use by default or for some particular fixture path.

Closing the issue waiting for ODM support.
Thanks!
